h/util/uri.py

```diff
@@ -291,3 +291,22 @@ def _normalize_queryvalue(value):
 def _blacklisted_query_param(s):
     """Return True if the given string matches any BLACKLISTED_QUERY_PARAMS."""
     return any(re.match(patt, s) for patt in BLACKLISTED_QUERY_PARAMS)
+
+
+def render_uri_template(template, values):
```

Comment:
Changed the name to 'uri' to match the file, but everything in here is actually specific to URLs...